### ISSUE TITLE
Please document python-xlib UnicodeDecodeError problem

### ISSUE THREAD

**Issue by alphapapa** [Khroki/MCEdit-Unified#883]
Hi,

I just ran into https://github.com/Khroki/MCEdit-Unified/issues/365:

```
[   ERROR][                   root.py:951]:MCEdit version Unified v1.5.6.0 %s Minecraft 1.8-1.11.2 | PE 0.11
Traceback (most recent call last):
  File "mcedit.py", line 1014, in <module>
    main(sys.argv)
  File "mcedit.py", line 948, in main
    MCEdit.main()
  File "mcedit.py", line 777, in main
    ('MCEdit ~ ' + release.get_version() % _("for")).encode('utf-8'), 'MCEdit'))
  File "/home/me/tmp/src/MCEdit-Unified/utilities/gl_display_context.py", line 24, in __init__
    self.reset(splash, caption=caption)
  File "/home/me/tmp/src/MCEdit-Unified/utilities/gl_display_context.py", line 64, in reset
    self.win = mcplatform.WindowHandler(mode=self.displayMode())
  File "/home/me/tmp/src/MCEdit-Unified/mcplatform.py", line 829, in __init__
    dis = self.display = Xlib.display.Display()
  File "/usr/lib/python2.7/dist-packages/Xlib/display.py", line 80, in __init__
    self.display = _BaseDisplay(display)
  File "/usr/lib/python2.7/dist-packages/Xlib/display.py", line 62, in __init__
    display.Display.__init__(*(self, ) + args, **keys)
  File "/usr/lib/python2.7/dist-packages/Xlib/protocol/display.py", line 61, in __init__
    name, host, displayno)
  File "/usr/lib/python2.7/dist-packages/Xlib/support/connect.py", line 93, in get_auth
    return getattr(mod, modname).get_auth(sock, dname, host, dno)
  File "/usr/lib/python2.7/dist-packages/Xlib/support/unix_connect.py", line 116, in new_get_auth
    au = xauth.Xauthority()
  File "/usr/lib/python2.7/dist-packages/Xlib/xauth.py", line 84, in __init__
    self.entries.append((family, addr.decode('UTF-8'),
  File "/usr/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xc0 in position 0: invalid start byte
```
I'm on Ubuntu Trusty 14.04, using the system `python-xlib` package.  After uninstalling that and installing it with `pip install --user python-xlib`, it worked.

It would be nice if this were documented so I didn't have to dig through the bug tracker to find the solution.  :)

Thanks for your work on MCEdit.

**Comment by LaChal** [comment]
This issue is know for several years now, and was solved in the installer, and already documented in the README right before the `BUILDING NBT AND PNG SUPPORT` part.

The installer is able to find most of the Python dependencies, and install them in the user profile only (to not break system application which may rely on another library version).
Even running from source, MCEdit will automatically use these user installed libraries.
The needed version for Python Xlib package is 0.14. I noticed some time ago that some Linux distros told they install this version, but actually the 0.12 one was installed.
This package is also included in the installer (unwanted output string were commented in this one, no other difference with the official code). It is pure python code, so unpacking the archive in `~/.local/lib/pyton2.7/site-packages` will work (actually what does the installer, and `pip --user`).
